docs/user-guide/FAQ.md

```diff
@@ -227,3 +229,7 @@ To resolve this issue, you could disable or tune the shallow clone settings ([se
 ## Why do I get `Not Found` when I try to start or delete my pipeline?
 
 Screwdriver pipelines have a 1:1 relation to the underlying SCM repository and this is validated by using not only the SCM repository name but also its unique repository ID. The `Not Found` error occurs when the SCM repository is deleted and re-created under same repository name (delete & re-fork has same effect). This action results in the new SCM repository getting a new ID and hence failing Screwdriver validations. If your pipeline is in this state, your only option is to re-create the pipeline and reach out Screwdriver cluster admin to remove the old pipeline.
+
+## How do I override freeze windows to start a build?
```

Comment:
doesn't UI does this already ? or is this for commits to ignore freeze window straightaway ?

Reply:
yeah I forgot the UI already does this
but it could be useful for people using the API or for commits

what do you think? I can close the PR if you don't think that's necessary

Reply:
I think just add a clarification that, if done by UI it's already handled and user can select the UI check mark to do this, but if done manually then add the message in commit.

Reply:
added